Source/Scene/Model.js

```diff
@@ -1077,8 +1079,35 @@ define([
                             var position = primitives[m].attributes.POSITION;
                             if (defined(position)) {
                                 var accessor = gltfAccessors[position];
-                                var aMin = Cartesian3.fromArray(accessor.min, 0, aMinScratch);
-                                var aMax = Cartesian3.fromArray(accessor.max, 0, aMaxScratch);
+                                var extensions = accessor.extensions;
+                                var accessorMin = new Matrix(accessor.min);
+                                var accessorMax = new Matrix(accessor.max);
+                                
+                                // If this accessor is quantized, we should use the decoded min and max
+                                if (defined(extensions)) {
+                                    var quantizedAttributes = extensions.WEB3D_quantized_attributes;
+                                    if (defined(quantizedAttributes)) {
```

Comment:
Okay, I'll post in the glTF thread for this and put together a pull request.

> If any extension spec decisions are holding this PR up, let's turn our attention to the gltf-pipeline implementation.

I still have one more issue on the Cesium side that I'm working out, then I'll focus on the pipeline.